views/default/object/comment.php

```diff
@@ -33,19 +33,33 @@
 		$body = elgg_view('output/longtext', [
 			'value' => $comment->description,
 		]);
+	
+		if (elgg_extract('show_add_form', $vars, true) && $comment->canComment()) {
+			$body .= elgg_view('output/url', [
+				'text' => elgg_echo('generic_comments:add'),
+				'href' => "#elgg-form-comment-save-{$comment->guid}",
+				'rel' => 'toggle',
+				'class' => 'elgg-subtext',
+			]);
+		}
 	}
 
 	$params = [
 		'icon_entity' => $commenter,
 		'time_href' => $comment->getURL(),
 		'access' => false,
 		'title' => false,
+		'show_summary' => true,
 		'content' => $body,
 		'imprint' => elgg_extract('imprint', $vars, []),
 		'class' => elgg_extract_class($vars),
 	];
 	$params = $params + $vars;
 	
+	if (!empty(elgg()->thread_preloader->getChildren($comment->guid))) {
```

Comment:
wouldn't it be nicer if the preloader has a function `hasChildren()`

Reply:
it is nicer, but for now i will not add it as it is Friday